packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -308,6 +308,14 @@ NOTE: this is ${styleBold.wrap('not')} recommended. Asset changes cannot be incl
     final releases = await codePushClientWrapper.getReleases(
       appId: appId,
     );
+
+    if (releases.isEmpty) {
+      logger.warn(
+        '''No releases found for app $appId. You need to make first a release before you can create a patch.''',
+      );
+      throw ProcessExit(ExitCode.success.code);
```

Comment:
@bryanoltman I was on the fence when choosing the code here, I ended up returning success because this is an expected error, but I wonder if `usage` would be more fitting, since in a way, the command was used wrongly?

Reply:
`usage` sounds like a better option.